### ISSUE TITLE
Unable to customize WalletMultiButton text

### ISSUE THREAD

**Issue by aaronpon** [solana-labs/wallet-adapter#88]
**Describe the bug**

According to the docs I should be able to change the button text by using

```
<WalletMultiButton> 
Connect 
</WalletMultiButton>
```

**To Reproduce**

Use 
```
<WalletMultiButton> 
Connect 
</WalletMultiButton>
```
to change the text of the buttons

**Expected behavior**
The button text should change

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by jordansexton** [comment]
Thanks! Looking into this.

**Comment by jordansexton** [comment]
I see what the problem is -- https://github.com/solana-labs/wallet-adapter/blob/c64705db3523bd3e3da79bd06355b23e30add8de/packages/ui/react-ui/src/WalletMultiButton.tsx#L59-L60

The `children` prop is only used as the button content when the wallet is connected.

**Comment by jordansexton** [comment]
Fixed by c419043080bfd667fc3f5405b6a05b6a01284232. Published:

 - @solana/wallet-adapter-ant-design@0.7.1
 - @solana/wallet-adapter-material-ui@0.12.1
 - @solana/wallet-adapter-react-ui@0.4.1
 
Please let me know if this fixes the issue for you. Keep in mind that if you are passing children to the button, it will override what is displayed in all 3 states (no wallet selected, wallet selected but not connected, wallet connected). You will need to account for this in the parent component.
